gix-diff/tests/diff/blob/slider.rs

```diff
@@ -1,6 +1,5 @@
-use gix_diff::blob::intern::TokenSource;
-use gix_diff::blob::unified_diff::ContextSize;
-use gix_diff::blob::{Algorithm, UnifiedDiff};
+use gix_diff::blob::v2::{Algorithm, BasicLineDiffPrinter, Diff, InternedInput, UnifiedDiffConfig};
```

Comment:
It's interesting that `v2` is available at all as it shouldn't be available by default.
Let me fix that

Reply:
Hmm, seems like this code should not compile, it's fine on `main`.

Reply:
Sorry about that, I didn't realise that this is a test in `gix-diff-tests`, which enables the feature. It was a long day yesterday 😅.

Reply:
No worries, I also could have provided that context. :-)